src/main/kotlin/service/RacingCarGameService.kt

```diff
@@ -23,31 +23,15 @@ class RacingCarGameService {
         return carsInfo
     }
 
-    fun getMaxPositionCars(carsInfo: List<Car>): List<Car> {
+    fun getMaxPositionCarNames(carsInfo: List<Car>): List<String> {
         val maxCarInfo = carsInfo.maxWith(Car::compareTo)
-        return carsInfo.filter { maxCarInfo.isSamePosition(it) }
-    }
-
-    fun getRoundResult(carsInfo: List<Car>): String {
-        var roundResult = ""
-        carsInfo.forEach { car ->
-            roundResult += "${car.getPresentStatus()}\n"
-        }
-        return roundResult
-    }
-
-    fun getWinners(carsInfo: List<Car>): String {
-        var winners = "최종 우승자: "
-
-        carsInfo.forEach { car ->
-            winners += "${car.name}, "
+        return carsInfo.filter { maxCarInfo.isSamePosition(it) }.map {
+            car ->
+            car.name
         }
```

Comment:
```suggestion
        return carsInfo.filter { maxCarInfo.isSamePosition(it) }
            .map { car ->
                car.name
            }
```
람다같은 경우에는 "변수명 ->" 까지는 같은 라인에 작성하는 것이 좋습니다
ref. https://kotlinlang.org/docs/coding-conventions.html#lambdas